sdk/apimanagement/apimanagement-custom-widget-scaffolder/review/apimanagement-custom-widget-scaffolder.api.md

```diff
@@ -0,0 +1,38 @@
+## API Report File for "@azure/apimanagement-custom-widget-scaffolder"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public
+export function generateProject(widgetConfig: TCustomWidgetCommonConfig, deployConfig: TDeployConfig, miscConfig?: TMiscConfig): Promise<void>;
+
+// @public
+export const OVERRIDE_DEFAULT_PORT = 3000;
+
+// @public
+export const OVERRIDE_PORT_KEY = "MS_APIM_CW_localhost_port";
+
+// @public
+export interface TCustomWidgetCommonConfig {
```

Comment:
We should remove `T` from all the interfaces

Reply:
well, it's still a type so I'm usually keeping the prefix anyway, but I agree with you as well, so let's use `I` prefix like `ICustomWidgetCommonConfig`?

Reply:
We don't prefix interfaces or types in our libraries, while it is a common practice in other languages, it is not in TS